androidApp/src/main/kotlin/org/mifos/mobile/HomeActivity.kt

```diff
@@ -47,6 +50,9 @@ class HomeActivity : ComponentActivity() {
     @Inject
     lateinit var networkMonitor: NetworkMonitor
 
+    @Inject
+    lateinit var preferenceHelper: PreferencesHelper
```

Comment:
Don't Inject `PreferenceHelper` in main activity, inherit it on `HomeActivityViewModel`

Reply:
And combine the theme state into `HomeActivityUiState` and collect it on Main activity and used it